### PR TITLE
 [bitnami/apache] Allow custom htdocs

### PR DESCRIPTION
**Pull request by javsalgar**
**Description of the change**
This PR allows mounting a custom static page. This is essential for the chart usability. There are different ways

   - Existing PVC
   - ConfigMap
   - Git repository 

**Benefits**

**Possible drawbacks**

**Applicable issues**

**Additional information**

### CODE REVIEW

**Comment by carrodher** [comment]
👋  @javsalgar can you resolve the conflicts before continuing?

**Comment by javsalgar** [comment]
Done, could you please check? @juan131 @tompizmor

File: bitnami/apache/README.md [hunk 1]
```diff
@@ -44,41 +44,53 @@ The command removes all the Kubernetes components associated with the chart and
...
+|----------------------------------|---------------------------------------------------------|--------------------------------------------------------------|
+| `global.imageRegistry`           | Global Docker image registry                            | `nil`                                                        |
+| `global.imagePullSecrets`        | Global Docker registry secret names as an array         | `[]` (does not add image pull secrets to deployed pods)      |
+| `image.registry`                 | Apache Docker image registry                            | `docker.io`                                                  |
+| `image.repository`               | Apache Docker image name                                | `bitnami/apache`                                             |
+| `image.tag`                      | Apache Docker image tag                                 | `{TAG_NAME}`                                                 |
+| `image.pullPolicy`               | Apache Docker image pull policy                         | `Always`                                                     |
```
Comment (juan131) [inline bitnami/apache/README.md]: We're using `IfNotPresent` since we moved to tags including the revision.

File: bitnami/apache/README.md [hunk 2]
```diff
@@ -44,41 +44,53 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `image.pullSecrets`              | Specify Docker registry secret names as an array        | `[]` (does not add image pull secrets to deployed pods)      |
+| `git.registry`                   | Git image registry                                      | `docker.io`                                                  |
+| `git.repository`                 | Git image name                                          | `bitnami/git`                                                |
+| `git.tag`                        | Git image tag                                           | `{TAG_NAME}`                                                 |
+| `git.pullPolicy`                 | Git image pull policy                                   | `Always`                                                     |
```
Comment (juan131) [inline bitnami/apache/README.md]: Same comment as above.

File: bitnami/apache/README.md [hunk 3]
```diff
@@ -44,41 +44,53 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `htdocsPVC`                      | PVC with the server static content                      | `nil`                                                        |
+| `cloneHtdocsFromGit.enabled`     | Get the server static content from a git repository     | `false`                                                      |
+| `cloneHtdocsFromGit.repository`  | Repository to clone static content from                 | `nil`                                                        |
+| `cloneHtdocsFromGit.branch`      | Branch inside the git repository                        | `nil`                                                        |
+| `cloneHtdocsFromGit.interval`    | Interval for sidecar container pull from the repository | `60`                                                         |
+| `podAnnotations`                 | Pod annotations                                         | `{}`                                                         |
+| `ingress.enabled`                | Enable ingress controller resource                      | `false`                                                      |
```
Comment (juan131) [inline bitnami/apache/README.md]: I'm missing the `resources`, `livenessProbe` and `readinessProbe` parameters information which are present in the **values.yaml**

File: bitnami/apache/README.md [hunk 4]
```diff
@@ -98,12 +110,33 @@ $ helm install --name my-release -f values.yaml bitnami/apache
...
+## Deploying your custom web application
+The Apache chart allows you to deploy a custom web application using one of the following methods:
+
+  - Cloning from a git repository: Set `cloneHtdocsFromGit.enabled` to `true` and set the repository and branch using the `cloneHtdocsFromGit.repository` and  `cloneHtdocsFromGit.branch` parameters. A sidecar will also pull the latest changes in an interval set by `cloneHtdocsFromGit.interval`.
+  - Providing a ConfigMap: Set the `htdocsConfigMap` value to mount a ConfigMap in the Apache htdocs folder. 
+  - Using an existing PVC: Set the `htdocsPVC` value to mount an PersistentVolumeClaim with the web application content.
+
+In the following example you can deploy a example web application using git:
+
+```
```
Comment (juan131) [inline bitnami/apache/README.md]: Use: ```console

**Pull request merged**